node/service/src/service.rs

```diff
@@ -536,3 +552,172 @@ pub async fn start_parachain_node(
 	)
 	.await
 }
+fn frequency_dev_instant(config: Configuration) -> Result<TaskManager, sc_service::error::Error> {
```

Comment:
function that runs authorship as manual seal and starts frequency as standalone not needing validators